src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/LocalizationModelDeserializer.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Orchestrator.RunnableProjects.Localization;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
+{
+    internal static class LocalizationModelDeserializer
+    {
+        private const string PostActionIndexPrefix = "postActions[";
```

Comment:
should those to be defined in localizer to be in sync? Or vice versa use those from localizer?

Reply:
These things are hidden behind `KeyCreator`s, so there is no const string like this in the localizer. I was initially planning to share some assembly between localizer and TE, but couldn't find a meaningful way to do it.